README.md

```diff
@@ -135,7 +135,8 @@ During preprocessing, the entities and relation types had their identifiers conv
 torchbiggraph_export_to_tsv \
   --dict data/FB15k/dictionary.json \
   --checkpoint model/fb15k \
-  --out joined_embeddings.tsv
+  --out-ent joined_embeddings.tsv
+  --out-rel joined_embeddings.tsv
```

Comment:
I prefer to avoid abbreviations, as I think they make code and commands a bit more cryptic, and I think a bit of added verbosity isn't too bad a thing. Could you rename these `--entities-output` and `--relation-types-output`?

Reply:
Why relation-types and not just relations?

Reply:
Because this is the naming convention we're using. "Relations" could be intended as edges, so we add "types" to make it clear that it is not each individual link but rather their classes. If you're wondering why we don't use "edge types", that's just historical reasons.